src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -570,6 +579,30 @@ private void WriteResult(TransactionOperationContext context, DynamicJsonValue d
                 }
             }
         }
+
+        [RavenAction("/operations/state", "GET", AuthorizationStatus.ValidUser)]
```

Comment:
Here are things that users will probably want to use (bulk insert start, for example), we can keep it active for all, but we need to check that we don't give you operations for dbs that you don't have access to).